docusaurus/docs/dev-docs/backend-customization/models.md

```diff
@@ -619,6 +619,10 @@ Lifecycle hooks can be customized declaratively or programmatically.
 Lifecycles hooks are not triggered when using directly the [knex](https://knexjs.org/) library instead of Strapi functions.
 :::
 
+:::tip
+Check our [error handling](/dev-docs/error-handling#services-and-models-lifecycles) documentation to resolve any errors you may encounter while using the lifecycle hooks.
```

Comment:
```suggestion
Please refer to the [error handling](/dev-docs/error-handling#services-and-models-lifecycles) documentation to learn how to throw errors from lifecycle hooks.
```

Reply:
The docs are not about resolving errors from lifecycles but about how to use the lifecycle hooks (methods) to actually throw errors.